pkg/apimachinery/tests/apis_meta_v1_unstructed_unstructure_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unstructured_test
+package tests
```

Comment:
> What about moving testapi as well? Then not much is left for these tests to be moved (api.List and Pod).

I think `testapi` needs a lot of work.  I'd prefer to avoid moving it.